ledger/participant-integration-api/src/main/scala/platform/store/DbType.scala

```diff
@@ -51,7 +51,6 @@ object DbType {
       sys.error(s"JDBC URL doesn't match any supported databases (h2, pg, oracle)")
   }
 
-  // TODO append-only: adapt AsyncCommit related configuration here
```

Comment:
stale, async commit for Oracle still not came. Removing as current setup with Async Commit configuration is appropriate for Postgres (which is the only one supports it)